src/index.js

```diff
@@ -149,7 +149,7 @@ class DAPIClient {
     const urlToConnect = await this.getGrpcUrl();
 
     const client = new CorePromiseClient(urlToConnect);
-    client.setProtocolVersion(version);
+    client.setProtocolVersion(protocolVersion);
```

Comment:
Should be a string, not an integer?

Reply:
@shumkov `package.json` would return string isn't it?

Reply:
And we use semver to check it

Reply:
Yeah, we changed this part in the common lib. It's good now.